README.md

```diff
@@ -3,54 +3,134 @@
+
+1. Push your working branch to your fork:  
+       $ git push -u origin <branch_name>
+1. From the repository page in GitHub, click **New pull request**.
+1. Select your working branch from the compare list.
+1. Add `WIP` to the title of the pull request.
+1. Add the **awaiting tech review** label to the pull request.
+1. In the pull request comment field, enter `@redhataccess/pantheon-developers Please review for technical completeness and accuracy`.
+1. Click **Create new pull request**.
+
+For code pull requests, one or more developers review the pull request. For documentation pull requests, the developers review the pull request for technical accuracy and documentation team members review the pull request for clarity and consistency and compliance with necessary standards.
+
+### The review process
+
+Both the technical review and peer review processes take place in pull requests in GitHub.
+
+After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review passed**.
+
+When technical review is complete, writers click the **Reviewers** gear icon and select the name of a team member to request peer review. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the reviewer clicks **Review changes > Approve** from the **Files changed** tab of the pull request to approve the changes and the pull request.
+
+### Merging a pull request
+
+When you have addressed all technical review and peer review comments, notify the developers to accept the pull request.
+
+1. Remove `WIP` from the title of the pull request.
+1. Click **Request Review** and enter `@redhataccess/pantheon-developers`.
+
+The developers check that the **Tech review passed** label has been added to the pull request and peer pull request approval provided, then accept it.
+
+## Installing Pantheon
+
+### Prerequisites
+* Podman
+* Buildah
+* Java
+
+### Building the application
+_(All commands from here on will be assumed to be run from the project's root directory)_
 
 ```sh
 ./mvnw clean install
 ```
 
-### Unit Tests
+### Unit tests
 
 ```sh
 ./mvnw test
 ```
 
-### Run the application
+### Running the application
 
 The best way to run Pantheon is to install [podman](https://podman.io).
 
-First, create a pod:
-
+1. First, create a pod:  
 ```sh
 podman pod create --name pantheon -p 8080 -p 5005
 ```
-
 This will create a `pantheon` pod with ports 8080 (for web access) and 5005 (for
 remote Java debugging) open.
 
-Run a mongo database container in the pod.
-
+1. Run a mongo database container in the pod.
 ```sh
 podman run --pod pantheon --name slingmongo -d mongo
 ```
-
-Build the pantheon docker image
-
+1. Build the pantheon docker image
```

Comment:
"Build the Pantheon Docker image"

Would it be more accurate to say "Docker-formatted container image"?

Reply:
Changed to: 1. 
> Build the Pantheon Docker-formatted container image.